fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -432,10 +437,56 @@ case $DistortionCorrection in
                 --smoothingfwhm="2" \
                 --inputdir="$WD"
         fi
-            
+
 
         ;;
 
+    ${NONE_METHOD_OPT})
+
+            log_Msg "---> No distortion correction"
+
+            log_Msg "---> FAKE WarpField"
+            ### faked warp image with three frames
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 ${WD}/WarpField.nii.gz
+            ${FSLDIR}/bin/fslmerge -t ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz
+
+            log_Msg "---> FAKE Jacobian Volume"
+            ###Jacobian Volume FAKED for Regular Fieldmaps (all ones) ###
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 -add 1 -bin ${WD}/Jacobian.nii.gz
+
+            log_Msg "---> FAKE spline interpolated image of scout"
+            # create a spline interpolated image of scout (distortion corrected in same space)
+            ${FSLDIR}/bin/imcp ${ScoutInputName} ${WD}/${ScoutInputFile}_undistorted
+
+            log_Msg "---> REORIENT scout to standard MNI152 template"
+            # use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images (MNI152)
+            fslreorient2std ${WD}/${ScoutInputFile}_undistorted.nii.gz ${WD}/${ScoutInputFile}_undistorted_fMRI2str.nii.gz
+            rm ${WD}/${ScoutInputFile}_undistorted.nii.gz
+            mv ${WD}/${ScoutInputFile}_undistorted_fMRI2str.nii.gz ${WD}/${ScoutInputFile}_undistorted.nii.gz
+
+            log_Msg "---> register scout image to T1w"
+            # register undistorted scout image to T1w
```

Comment:
Consistent with my above comments, this comment `# register undistorted scout image to T1w` is apparently just a copy/paste from the SEFM correction case, and in fact the comment is completely wrong in this context. (The file may have "_undistorted" in its file name, but it is in fact the original (distorted) scout.